package.json

```diff
@@ -34,7 +34,7 @@
     "ios:production-release": "react-native run-ios --scheme MindloggerMobile --configuration Release"
   },
   "dependencies": {
-    "@georstat/react-native-image-cache": "^2.3.0",
+    "@georstat/react-native-image-cache": "^3.1.0",
     "@hookform/resolvers": "^2.9.10",
     "@miblanchard/react-native-slider": "^2.3.1",
```

Comment:
Please, notify to Natalia of possible regressions related to changed library versions